gm4_blast_furnaces/data/blast_furnaces/functions/create_furnace_output.mcfunction

```diff
@@ -0,0 +1,7 @@
+# Called from recipe_check.mcfunction
+
+# ============== CONTAINS THE CUSTOM CRAFTERS EXTENSION WHERE THE CRAFTING RECIPE IS CHECKED ================
+setblock ~ ~ ~ hopper{CustomName:"{\"text\":\" Blast Furnace Output\"}"}
+summon minecraft:armor_stand ~ ~ ~ {NoGravity:1b,Marker:1b,Small:1b,Tags:["gm4_bf_output"],Invisible:1b}
+playsound minecraft:block.anvil.use block @a[distance=..10] ~ ~ ~ 5 2
+kill @s
```

Comment:
Is there any reason to summon an entirely new armor_stand? Is it more efficient than modifying the old one or does it require a new position?

Reply:
The reason a lot of inconsistencies exist is cause I started from @iDoomfull's code. The === comments where his which I hadn't changed

This is also an exampke of that. I'll see what tags have to be changed to allow for keeps ng the old enemy.